Frameworks/Foundation/NSOperationQueue.mm

```diff
@@ -330,132 +424,43 @@ - (void)addOperations:(id)operations waitUntilFinished:(BOOL)wait {
 */
 - (void)setMaxConcurrentOperationCount:(NSInteger)count {
     UNIMPLEMENTED();
-    priv->_maxConcurrentOperationCount = count;
+    _priv->SetMaxConcurrentOperationCount(count);
```

Comment:
not in spirit. They are implemented in that they don't break things but they do not actually influence concurrency. I decided to leave as is. The re-write will actually do something useful there presumably.

---
In reply to: [95449162](https://github.com/Microsoft/WinObjC/pull/1676#discussion_r95449162) [](ancestors = 95449162)